scene/2d/tile_map.cpp

```diff
@@ -1707,6 +1819,12 @@ void TileMap::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("set_y_sort_mode", "enable"), &TileMap::set_y_sort_mode);
 	ClassDB::bind_method(D_METHOD("is_y_sort_mode_enabled"), &TileMap::is_y_sort_mode_enabled);
 
+	ClassDB::bind_method(D_METHOD("set_compatibility_mode", "enable"), &TileMap::set_compatibility_mode);
+	ClassDB::bind_method(D_METHOD("is_on_compatibility_mode"), &TileMap::is_on_compatibility_mode);
```

Comment:
I would make the getter `is_compatibility_mode_enabled` to follow the same style as other getters.